src/Sentry/Internal/PlatformEventProcessor.uwp.cs

```diff
@@ -0,0 +1,82 @@
+#if WINDOWS_UWP
+using System;
+using Sentry.Extensibility;
+using Windows.ApplicationModel;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.System.Profile;
+
+namespace Sentry.Internal
+{
+
+    internal class PlatformEventProcessor : ISentryEventProcessor
+    {
+        private Lazy<UwpContext> _uwpContext = new Lazy<UwpContext>(() => new UwpContext());
+        private volatile bool _uwpContextLoaded = true;
+        private class UwpContext
+        {
+            public string DeviceFamily { get; set; }
```

Comment:
Couldn't these be readonly?

```suggestion
            public string DeviceFamily { get; }
```